### ISSUE TITLE
Examples in docs may be outdated

### ISSUE THREAD

**Issue by mjuric** [dirac-institute/sorcha#1065]
While implementing the cmdline option update PR, I noticed some of the docs may be out of date. E.g. https://sorcha.readthedocs.io/en/latest/gettingstarted.html shows the old `sorcha --help` result (even before we switched to `sorcha <verb>` scheme).

I'm not sure how these are (auto-?) generated & I'm worried I'll break something if I go and manually change the example files, so I'm just making a note here for someone to scrub the docs before v1.0:

<img width="715" alt="image" src="https://github.com/user-attachments/assets/a7868506-7bcc-4dab-99d8-fc657ceba4a9">

**Comment by mschwamb** [comment]
yep docs haven't been finished. We're waiting for the linux conversion of commands before fixing things
